bookshelf/requirements.txt

```diff
@@ -1,7 +1,7 @@
 Flask==2.0.1
 google-cloud-firestore==2.1.1
 google-cloud-storage==1.38.0
-google-cloud-logging==2.4.0
+google-cloud-logging==2.3.1
```

Comment:
Please indicate that google-cloud-logging should be <2.4 and >=1.14.0 in a comment format.